released/packages/coq-bonsai/coq-bonsai.1.0.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "2.0"
+maintainer: "dev@clarus.me"
+homepage: "https://github.com/foobar-land/coq-bonsai"
+dev-repo: "git+https://github.com/foobar-land/coq-bonsai.git"
+bug-reports: "https://github.com/foobar-land/coq-bonsai/issues"
+authors: ["Guillaume Claret"]
+license: "GPL-3.0"
```

Comment:
is this `GPL-3.0-only` or `GPL-3.0-or-later`? Unfortunately `GPL-3.0` is not an SPDX identifier.

Reply:
OK, I did the change to `GPL-3.0-only`.